nautobot/dcim/models/device_components.py

```diff
@@ -351,6 +353,8 @@ def get_power_draw(self):
 
             # Calculate per-leg aggregates for three-phase feeds
             if getattr(self._cable_peer, "phase", None) == PowerFeedPhaseChoices.PHASE_3PHASE:
+                # Setup numerator and denominator for 
```

Comment:
...for what? :-)

Reply:
For the love of numerating and denominating :)